googletest/crate_docs.md

```diff
@@ -142,6 +142,8 @@ The following matchers are provided in GoogleTest Rust:
 | [`has_entry`]        | A [`HashMap`] containing a given key whose value the argument matches.   |
 | [`is_contained_in!`] | A container each of whose elements is matched by some given matcher.     |
 | [`is_nan`]           | A floating point number which is NaN.                                    |
+| [`is_finite`]        | A floating point number which is neither infinite nor NaN.               |
+| [`is_infinite`]      | A floating point number which is positive or negative infinity.          |
```

Comment:
I think you need to add a link target to the block below (see line 166 and below).